traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java

```diff
@@ -184,11 +184,18 @@ public static AuthenticatedUrlRequest cancellationQuote(Order order, Session ses
                 .build(session.getToken());
     }
 
-    public static AuthenticatedUrlRequest cancelOrder(CancellationQuote quote, Session session, Context context) {
-        return new RouteBuilder(context, session.getApiKey())
+    public static AuthenticatedUrlRequest cancelOrder(CancellationRequest cancellationRequest, Session session, Context context) {
+        CancellationQuote quote = cancellationRequest.getCancellationQuote();
+        CancellationReason reason = cancellationRequest.getCancellationReason();
+        RouteBuilder rb = new RouteBuilder(context, session.getApiKey())
                 .method(NetworkTask.Route.Method.PATCH)
                 .path("/order/" + quote.getOrder().getId() + "/cancellation/" + quote.getId())
-                .build(session.getToken());
+                .param("cancellationReason", cancellationRequest.getCancellationReason().getId());
```

Comment:
I just noticed this on iOS as well.

only GET requests should get input from query params. This needs to be in the body.

Future concerns like this should be brought up with the back end dev working on the feature

Reply:
It's handled in backend, so you can proceed with the update.

Reply:
I've updated PR